paper/joss/reconst_ivim/paper.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Ivim: a model for diffusion and perfusion analysis of MRI'
+tags:
+  - dipy
+  - ivim
+  - diffusion
+  - perfusion
+  - intravoxel incoherent motion
+  - MRI
+  - bi-exponential fitting
+authors:
+ - name: Shahnawaz Ahmed
+   orcid: 0000-0003-1145-7279
+   affiliation: 1
+ - name: Ariel Rokem
+   orcid: 0000-0003-0679-1985
+   affiliation: 2
+ - name: Eric Peterson
+   orcid: 0000-0002-3530-9652
+   affiliation: 3
+ - name: Rafael Neto Henriques
+   orcid: 0000-0000-0000-1234
+   affiliation: 4
+affiliations:
+ - name: Birla Institute of Technology and Science Pilani
+   index: 1
+ - name: The University of Washington eScience Institute
+   index: 2
+ - name: SRI International
+   index: 3
+ - name: Cognition and Brain Sciences Unit (CBU)
+   index: 4
+date: 6 October 2016
+bibliography: paper.bib
+---
+
+# Summary
+
+Intra-voxel incoherent motion (IVIM) is a biophysical model that describes diffusion
+and perfusion in the signal acquired with a multi-diffusion value MRI
+acquisition [@le1986mr]. This method was originally developed to study the
+fast and slow motion of water molecules in the human brain -- an indication of brain
```

Comment:
How about: 

This method was originally developed to study the perfusion and diffusion in the human brain [@le1986mr], but more recently, interest in this technique has expanded ...